specs/lib/tipsifyGltfSpec.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var clone = require('clone');
+var tipsifyGltf = require('../../lib/tipsifyGltf');
+var addDefaults = require('../../lib/addDefaults');
+var readGltf = require('../../lib/readgltf');
+
+var gltfPath = './specs/data/tipsifyGltf/Cesium_Ground.gltf';
+
+describe('tipsifyGltf', function() {
+    it('reorders indices', function(done) {
+        readGltf(gltfPath, function(gltf) {
+            addDefaults(gltf);
+            var copyGltf = clone(gltf);
+            tipsifyGltf(gltf);
+
+            expect(gltf).not.toEqual(copyGltf);
```

Comment:
I was suspicious that this passed, and it looks like it always will. I commented the tipsify line and it passes, meaning clone is always changing something about the gltf. `expect(clone(gltf)).toEqual(copyGltf);` seems to be accurate though, so you should also make that change in `gltfPipelineSpec.js` and anywhere else that does that. For this particular test though, I would just compare the indices alone, not the whole gltf.